.changeset/blue-rocks-smoke.md

```diff
@@ -0,0 +1,18 @@
+---
+'astro': patch
+'@astrojs/netlify': patch
+'@astrojs/node': patch
+---
+
+Netlify Adapter
+
+This change adds a Netlify adapter that uses Netlify Functions. You can use it like so:
+
+```js
+import { defineConfig } from 'astro/config';
+import { netlifyFunctions } from '@astrojs/netlify';
```

Comment:
Super nit: I have a preference for a `default` export so users can call it whatever they want.

```js
import netlify from '@astrojs/netlify/functions';
```

Reply:
The reason I did it this way is because Netlify has 2 serverless hosting options and I thought each would be their own export. Could make this one the default for now, though.